CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Fixed
 - in PR template spell "compatibility" correctly
 
+## [2.1.1] - 2017-07-07
```

Comment:
Also this is technically a breaking change rather than a patch so it really needs to go under a `### Breaking Changes` header.